assets/upgradePatches.json

```diff
@@ -9,6 +9,20 @@
           "value": true
         }
       ]
+    },
+    {
+      "semverRange": ">=1.5.0 <=1.6.0",
```

Comment:
Not sure about these versions, can anyone confirm?

Reply:
```suggestion
      "semverRange": ">=1.4.0 <1.5.0",
```